plausible/docker-compose.yml

```
@@ -0,0 +1,49 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: plausible_1
+      APP_PORT: 9091
+
+  plausible_db:
+    # Plausible v2.1.1 was tested against PostgreSQL versions 15 and 16
+    # https://github.com/plausible/analytics/blob/v2.1.1/.github/workflows/elixir.yml#L21-L32
+    image: postgres:16-alpine@sha256:492898505cb45f9835acc327e98711eaa9298ed804e0bb36f29e08394229550d
+    restart: always
+    volumes:
+      - db-data:/var/lib/postgresql/data
```

For the volumes, please use the umbrelOS-provided bind mounts. Here is a suggestion
```suggestion
      - ${APP_DATA_DIR}/db-data:/var/lib/postgresql/data
```